DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Behavioiur : MonoBehaviour {
+    private GameObject player;
+    private Rigidbody2D rb;
+    public int thrust;
+    private CircleCollider2D circle;
+
+    // Use this for initialization
+    void Start () {
+        player = GameObject.FindGameObjectWithTag("Player");
+        rb = GetComponent<Rigidbody2D>();
+        circle = GetComponent<CircleCollider2D>();
+        thrust = 100;
```

Comment:
Yes take line 16 out. but initialize 100 in the inspector and then push the code so that other people do not have to initialize it themself for the code to work on their computer.